### PR TITLE
podmanv2 history and image remove templates

### PR DESCRIPTION
**Pull request by baude**
remove the use of template functions images and history to allow for straight-forward user experience.  instead of templates we use structs and struct methods.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
One small NIT,
LGTM
I like the simplification.

File: cmd/podmanV2/images/history.go [hunk 1]
```diff
@@ -100,24 +104,50 @@ func history(cmd *cobra.Command, args []string) error {
...
-	err = tmpl.Execute(w, results.Layers)
+	err = tmpl.Execute(w, hr)
 	if err != nil {
 		fmt.Fprintln(os.Stderr, errors.Wrapf(err, "Failed to print report"))
 	}
 	w.Flush()
 	return nil
 }
+
+type historyreporter struct {
```
Comment (mheon) [inline cmd/podmanV2/images/history.go:127]: Any reason we can't hang the methods off entities.ImageHistoryLayer?

File: cmd/podmanV2/images/list.go [hunk 2]
```diff
@@ -9,9 +9,11 @@ import (
 	"text/tabwriter"
 	"text/template"
 	"time"
+	"unicode"
+
+	"github.com/docker/go-units"
```
Comment (mheon) [inline cmd/podmanV2/images/list.go:15]: Extra newline

File: cmd/podmanV2/images/history.go [hunk 3]
```diff
@@ -100,24 +104,50 @@ func history(cmd *cobra.Command, args []string) error {
...
+}
+
+func (h historyreporter) CreatedBy() string {
+	if len(h.ImageHistoryLayer.CreatedBy) > 45 {
+		return h.ImageHistoryLayer.CreatedBy[:45-3] + "..."
+	}
+	return h.ImageHistoryLayer.CreatedBy
+}
+
+func (h historyreporter) ID() string {
```
Comment (mheon) [inline cmd/podmanV2/images/history.go:148]: Does `--no-trunc` affect this?

File: cmd/podmanV2/images/history.go (hunk 1 above)
Comment (baude) [reply cmd/podmanV2/images/history.go:127]: evidently the preference is that it be done like this to keep it encapsulated.

File: cmd/podmanV2/images/history.go (hunk 3 above)
Comment (baude) [reply cmd/podmanV2/images/history.go:148]: yup

**Pull request merged**